addons/kommander/1.163.x/kommander-2.yaml

```diff
@@ -0,0 +1,81 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: kommander
+  labels:
+    kubeaddons.mesosphere.io/name: kommander
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "1.163.2-2"
+    appversion.kubeaddons.mesosphere.io/kommander: "1.163.2"
+    endpoint.kubeaddons.mesosphere.io/kommander: /ops/portal/kommander/ui
+    appversion.kubeaddons.mesosphere.io/thanos: 0.3.9
+    appversion.kubeaddons.mesosphere.io/karma: 1.4.0
+    appversion.kubeaddons.mesosphere.io/kommander-grafana: 6.4.2
+    endpoint.kubeaddons.mesosphere.io/thanos: /ops/portal/kommander/monitoring/query
+    endpoint.kubeaddons.mesosphere.io/karma: /ops/portal/kommander/monitoring/karma
+    endpoint.kubeaddons.mesosphere.io/kommander-grafana: "/ops/portal/kommander/monitoring/grafana"
+    docs.kubeaddons.mesosphere.io/thanos: "https://thanos.io/getting-started.md/"
+    docs.kubeaddons.mesosphere.io/karma: "https://github.com/prymitive/karma"
+    docs.kubeaddons.mesosphere.io/kommander-grafana: "https://grafana.com/docs/"
```

Comment:
@gracedo it's kind of hard to see in the diff but all that's changed in these files are the annotations - specifically, the endpoints (changed to the `/ops/portal/kommander/monitoring/x` pattern per [slack](https://mesosphere.slack.com/archives/CJQURP1FT/p1578427926041100)), and adding annotations for kommander's grafana. Also changed the kommander UI endpoint as I think that the monitoring endpoint changes are dependent on the ui path change? Let me know if this looks right!